OIIInspector/OIIIClient.py

```diff
@@ -1,9 +1,38 @@
-import json
+from LocalExecutor import LocalExecutor
+from GrpCurlAddressImporter import GrpCurlAddressImporter
+import ast
 
 
 class OIIIClient:
 
-    def get_index_image_package_list(self, image_address, address_type):
+    def __init__(self):
+        self.grp_curl_position = GrpCurlAddressImporter().grp_curl_address
+        self.executor = LocalExecutor()
 
-        print(image_address, address_type)
-        return json.loads("{}")
+    def get_index_image_package_list(self, image_address):
+        command_to_call = '{grp_curl_position} {call_type} {image_address} {call_argument}'.format(
+            grp_curl_position=self.grp_curl_position,
+            call_type="-plaintext",
+            image_address=image_address,
+            call_argument="list")
+        out = self.executor.run_cmd(command_to_call, "Call failed to execute")
+        return out.strip("\n").split("\n")
+
+    def get_bundle(self, image_address):
+        command_to_call = '{grp_curl_position} {call_type} {image_address} {call_argument}'.format(
+            grp_curl_position=self.grp_curl_position,
+            call_type="-plaintext",
+            image_address=image_address,
+            call_argument="api.Registry/GetBundle")
+        out = self.executor.run_cmd(command_to_call, "Call failed to execute")
+        return out.strip("\n").split("\n")
+
+    def list_packages(self, image_address):
+        command_to_call = '{grp_curl_position} {call_type} {image_address} {call_argument}'.format(
+            grp_curl_position=self.grp_curl_position,
+            call_type="-plaintext",
+            image_address=image_address,
+            call_argument="api.Registry/ListPackages")
+        out = self.executor.run_cmd(command_to_call, "Call failed to execute")
+        out = out.replace("\n", "").replace("}", "},")
+        return ast.literal_eval(out)
```

Comment:
I think grpcurl outputs some pseudo-JSON. Though similar to Python expressions, I wouldn't count on it being translatable in all cases. I would prefer if you could transform the command output to a format that can be consumed by json.loads()